client/flutter/assets/content_bundles/mythbusters.en.yaml

```diff
@@ -0,0 +1,76 @@
+schema_version: 1
+content_version: 1
+contents:
+  type: mythbusters
```

Comment:
the type here and in the questions answered should be the same, but that's for another PR